### PR TITLE
Update paper messages.

### PR DESCRIPTION
**Pull request by youxingling**
### What changes were proposed in this pull request?

Update the latest paper message.

### Why are the changes needed?

Help users learn about our work.

### Does this PR introduce any user-facing change?

No.

### How was this patch tested?

CI.

### CODE REVIEW

File: docs/blogs/dlrover_rm.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+computing resources for the job, including the number of nodes and the resources of each node (CPU
+number & memory size, etc.). Users do not want to allocate too many resources, which will lead to
+resource wastage, and they are also worried about allocating too few resources, which will lead to
+failure of the training job. Currently, a large number of training jobs on kubernetes clusters
+suffer from irrational resource allocation and generally low utilization of computing resources.
+In addition, the high instability of cloud environments leads to high failure rates and frequent
+anomalies (e.g., slow nodes) in DLRM training.
+
+To this end, this paper develops a cloud-based deep learning training system specifically designed
+ for DLRM, which takes into account runtime training information to accurately allocate and
```
Comment (workingloong) [inline docs/blogs/dlrover_rm.md:17]: “runtime training information” -> "training runtime information"

File: docs/blogs/dlrover_rm.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+job's worker node (e.g., hosted in a pod) for training.
+
+<div align="center">
+<img src="../figures/dlrover_rm/sys_overview.png" alt="Editor" width="600">
+
+<text>Figure 1: Overview of DLRover and Model Training Workflow</text>
+</div>
+
+After submission of a job by the user, Cluster Brain quickly learns the characteristics of the job
+ and generates an initialized (warm-starting) resource plan by leveraging the relevant historical
```
Comment (workingloong) [inline docs/blogs/dlrover_rm.md:56]: The empty space is not need at the head of sentence in markdown.

File: docs/blogs/dlrover_rm.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+<div align="center">
+<img src="../figures/dlrover_rm/data_sharding.png" alt="Editor" width="600">
+
+<text>Figure 2: Dynamic Data Sharding</text>
+</div>
+
+### Seamless Migration
+
+DLRover develop a seamless migration mechanism that effectively reduces training downtime by
```
Comment (workingloong) [inline docs/blogs/dlrover_rm.md:115]: develop -> develops

Comment (youxingling) [reply docs/blogs/dlrover_rm.md]: Done.

**Review by workingloong — APPROVED** [review]
LGTM

**Pull request merged**